coinratecap/Backend
# makes job id dynamic , adds service , controller and route for exchange

## What this PR does ? 
- adds exchange service, exchange controller and exchange route
- makes job IDs for price fetch cron jobs dynamic
- removes `build` script from package.json which used to `seed` our db

### How to test the exchange end point ? 
- Go to url `http://localhost:3000/v1/exchange/all` from browser to see the end point response (all exchanges)
